### DISCUSSION TITLE
How do i increase memory limit?

### DISCUSSION THREAD

**Discussion by switchyfoot** [wolveix/satisfactory-server#194] [Q&A] [answered]
Friend and I have been playing on this docker container dedicated server (running in podman on Ubuntu, setup using the run CLI) and I noticed we are running up against the 16GB limit. How do I increase the memory limit now that I already have the container created?

**Comment by wolveix** [comment, marked as answer]
Hey @switchyfoot :) Just destroy and rebuild the container. If you've used a bind mount (as is recommended to save game files and data), then the container state doesn't matter

```
docker stop satisfactory-server
docker rm satisfactory-server
docker run ...
```

As a sidenote, I'd very strongly recommend switching to using Docker Compose. Makes the process of updating and such much easier :)
